PAD optional (#384)
biometricITC/cPP-biometrics: Supporting Documents/BS_SD.adoc

=== Introduction

> PAD is an optional set of requirements, when FIA_MBE_EXT.3 or FIA_MBV_EXT.3 are included in an evaluation. This set of Evaluation Activities are only performed if one (or both) of these requirements are included.

Reviewer: ```suggestion
PAD is an optional set of requirements, when FIA_MBE_EXT.3 or FIA_MBV_EXT.3 are included in an evaluation. This set of Evaluation Activities are only performed if any of these requirements are included.
```

Reviewer 2: That works for me.
